dod-advana/gamechanger-web
# Bugfix/uot-143789 - pre search filter bug

## Description

When there are pre search filters a section of the handleSearch tries to iterate over the organization and type filters with a forEach, but both of those are stored as an Object, not an array. This would throw an error and consequently the search would never finish loading and display results.

## !vibez

thumbs up

## Related Issue/Ticket

[UOT-142928](https://jira.di2e.net/browse/UOT-142928)

## Types of changes

## Smoke testing instructions

1. Clear browser cache
2. Go to gamechanger home
3. Click advanced search and select a type (like "Manual") and a source (like "Coast Guard")
4. Then enter a search term (like "raft") and search
5. Verify search results load correcctly!

This exact search prior to bug fix would load forever due to the object/array iterating error mentioned above.

## Checklist:
